### ISSUE TITLE
slow and steady  40 min run all is well

### ISSUE THREAD

**Issue by ca2014mp2** [finndev/PokeBuddy#987]
running for 43 min farming stops... no soft ban... no vpn. slow and steady at 5km/h which is normal human speed...

**Comment by shiitan5** [comment]
doesnt work ... i get ban every 2 min right after using the bot .... 40km/h or even 2km/h doesnt matter

**Comment by Synrise** [comment]
been running for 1hour and 50min at 7km/h, all tasks are on besides evolving pokémon, no use of incubators aswell. Getting 20k xp/hour

**Comment by Kokainium** [comment]
Im running for 2 hours and 33 min no problems:)

**Comment by ca2014mp2** [comment]
yeah my incubators have never worked..cept when I loaded them and then turned on bot.

**Comment by onilax** [comment]
![capture](https://cloud.githubusercontent.com/assets/20994804/17630862/f3ec469c-60f3-11e6-8f23-c4fce253bbd0.PNG)

**Comment by ca2014mp2** [comment]
low and slow seems to be the trick..

**Comment by onilax** [comment]
15km/hr.. planning to reduce it into 5-10 maybe..

**Comment by ca2014mp2** [comment]
damn... 15.... i get popped at 10...  running 5 now

**Comment by Kokainium** [comment]
My walkspeed is 20 :)

**Comment by ca2014mp2** [comment]
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

**Comment by Kokainium** [comment]
I have been walking 20 all the time :) everytime it stops i log on my Phone to see if im banned but nope :)

**Comment by ca2014mp2** [comment]
update... got popped - soft ban... system stopped and has not move forward.  map frozen.. restarting now.

**Comment by Sid733** [comment]
i started with 20 and its still 20km/hr 
never got ip ban or soft Ban

**Comment by stavrosian** [comment]
its so strange it works for some and not for others.
I tried 5 KPH, and got banned within 2 minutes.

**Comment by Kokainium** [comment]
Use vpn?

**Comment by Sid733** [comment]
no dnt hv that time to install vpn then set location .....i dnt care if my id is banned bcoz its nt even release in my country yet

**Comment by Kokainium** [comment]
I Takes 5 mins?

**Comment by Sid733** [comment]
excuse english

**Comment by Kokainium** [comment]
It takes

**Comment by Kokainium** [comment]
Where u from?

**Comment by Sid733** [comment]
for what?

**Comment by Sid733** [comment]
pakistan

**Comment by Kokainium** [comment]
Install vpn

**Comment by Kokainium** [comment]
Aarh ok

**Comment by Sid733** [comment]
which one? name or link

**Comment by Kokainium** [comment]
Softether VPN

**Comment by Sid733** [comment]
let mt check

**Comment by onilax** [comment]
those who got banned, are you using multiple bots?

**Comment by ca2014mp2** [comment]
one hour without issue before it froze up... soft ban for sure.. here's the thing.. my phone is fine.. logged back onto my account and could see stops and gyms... humm.. strange days for sure... more later

**Comment by ca2014mp2** [comment]
just using one bot here..

**Comment by Sid733** [comment]
i think i use  every bot right now i m using poke farmer for catching pokemons and pokebuddy for transfering them and recycling inventory

**Comment by Sid733** [comment]
at afternoon, when necrobot wrks i use it just for sniping pokemon and other two for same function as mention above at same time on 1 account

**Comment by imbot87** [comment]
I think that crashes when the game releases the warning popup that has passengers and not drivers, if you go to 5kh the problem should not be

**Comment by icozmo** [comment]
@ca2014mp2  If I may ask..  Are  you using vpn?
 Are the ip "you're using whether it's a vpn or no" and  the location you're farming in the same location?

**Comment by ca2014mp2** [comment]
I'm only using it locally for the past few days, not using vpn. Even while using it locally, and moving slow 5km/h and JUST getting stops... I'm getting banned. I seem to also be in the area of first roll out of changes,.. Silicon Valley CA.. San Jose to be exact. I should stress that in each time I will turn off the bot from my laptop and kick up my phone and the phone works fine.. I once or twice then restarted the bot but each time it didn't work until i let it rest for several hours.. (not tracking the rest time interval) and the bot will work for about an hour or so... went to set up vpn but need a host address.. will look for that tomorrow..(softether vpn) tried a ipconfig/realease - renew to bounce ip but tokens kept it the same.. vpn will prove it its targeting my ip...

**Comment by ca2014mp2** [comment]
more info... seems that just doing an ipconfig/release and renew will unjam  it. now running at 10k/h and when it locks up just do a release and renew and away it goes.. soooo strange..thoughts?

**Comment by ca2014mp2** [comment]
it WORKS... someone else please try and confirm it independently. did it 3x no issues..

**Comment by ca2014mp2** [comment]
1  hour run...
